ui/src/Logout.tsx

```diff
@@ -1,29 +1,32 @@
 // Libraries
-import {PureComponent} from 'react'
+import {FC, useEffect} from 'react'
 import {withRouter, WithRouterProps} from 'react-router'
+import auth0js from 'auth0-js'
 
 // APIs
 import {postSignout} from 'src/client'
+import {getAuth0Config} from 'src/authorizations/apis'
 
 // Constants
 import {CLOUD, CLOUD_URL, CLOUD_LOGOUT_PATH} from 'src/shared/constants'
 
 // Components
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
-type Props = WithRouterProps
-
-@ErrorHandling
-export class Logout extends PureComponent<Props> {
-  public componentDidMount() {
-    this.handleSignOut()
-  }
-
-  public render() {
-    return null
-  }
-
-  private handleSignOut = async () => {
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+const Logout: FC<WithRouterProps> = ({router}) => {
```

Comment:
you need some more types in here. try running this in `influxdb/ui`:
`yarn tsc -noImplicitAny | grep -A 3 -i "$(git diff --name-status master | awk '{ print substr($2,4) }')`